jenkins/open-mpi.dist.create-tarball.build-rpm.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+# usage: build-rpm.sh <build_prefix> <srpm_name>
+#
+# Script to build a binary RPM from a source RPM
+#
+# Expected filesystem layout:
+#    ${WORKSPACE}/ompi-scripts/         ompi-scripts checkout
+#    ${WORKSPACE}/ompi/                 ompi checkout @ target REF
+#    ${WORKSPACE}/dist-files/           output of build
+
+set -e
+
+build_prefix="$1"
+srpm_name="$2"
+
+aws s3 cp ${build_prefix}/${srpm_name} ${srpm_name}
+
+# Build and install PRRTE and OpenPMIX, if available
+if [[ -d ${WORKSPACE}/ompi/3rd-party/openpmix ]]; then
+    pushd ${WORKSPACE}/ompi/3rd-party/openpmix
```

Comment:
It would be good to print something like "Building OpenPMIx RPM", "Installing OpenPMIx RPM", etc. here, so that you can figure out where you are if something goes wrong.